### ISSUE TITLE
Thread/Post sorting 

### ISSUE THREAD

**Issue by auryn-macmillan** [auryn-macmillan/DAOForum#4]
**Which sorting options are available and which should be implemented**

**Comment by ghost** [comment]
I'm a big fan of the suggestions you outlined in the slack. 

-Weighted by stake
-Weighted by web of trust (complicated, but boils down to number of users endorsing a particular user, number of users endorsing the endorser, and so on and so forth) <-- very interesting if feasible
-Weighted by time (newest first/last)
-Weighted by up/down votes
-Weighted by controversy (total votes up or down)

Honestly I think that about covers it. Maybe add something like weighted by replies to view the comment chains with the most discussion?

**Comment by auryn-macmillan** [comment]
yeah weighted by replies is good too

**Comment by vmichalik** [comment]
also need time categories in case there are lots of proposals e.g. day, week and month. might even need categories for proposals, but not sure how those would look.

**Comment by auryn-macmillan** [comment]
Yes, sorted by most upvotes/downvotes/total votes/replies/etc in the past 24hours/week/month/year/arbitrary number of days/etc is a great idea.

On slack I mentioned the idea that perhaps new proposals could automatically generate a new thread, but perhaps they could actually generate a new sub-forum to house all discussion of the that proposal?

**Comment by ghost** [comment]
I second that new proposals should automatically generate an official thread. The distinction between a proposal thread and a discussion thread should be clear. Proposal threads should have additional functionality. Perhaps having one 'Main' section within the proposal thread (that can be sorted in the various ways above) and then unique subthreads for each article of the proposal? If there is only article that is particularly contentious, there's no need for it to take up all of the 'Main' discussion space.
